Orange/widgets/classify/tests/test_owsvmclassification.py

```diff
@@ -0,0 +1,120 @@
+from Orange.widgets.tests.base import GuiTest
+
+
+class TestOWSVMClassification(GuiTest):
+    def setUp(self):
+        self.widget = OWSVMClassification()
+        self.event_data = None
+
+    # Check if right text is written for specific kernel
+    @unittest.skip("Not corrected widget on load")
+    def test_kernel_equation_run(self):
+        if self.widget.kernel_box.buttons[0].isChecked():
+            self.assertEqual(self.widget.kernel_eq, self.widget.kernels[0][1])
+        elif self.widget.kernel_box.buttons[1].isChecked():
+            self.assertEqual(self.widget.kernel_eq, self.widget.kernels[1][1])
+        elif self.widget.kernel_box.buttons[2].isChecked():
+            self.assertEqual(self.widget.kernel_eq, self.widget.kernels[2][1])
+        elif self.widget.kernel_box.buttons[3].isChecked():
+            self.assertEqual(self.widget.kernel_eq, self.widget.kernels[3][1])
+
+    # Check if right text is written for specific kernel after click
+    def test_kernel_equation(self):
+        self.widget.kernel_box.buttons[0].click()
+        self.assertEqual(self.widget.kernel_eq, self.widget.kernels[0][1])
+        self.widget.kernel_box.buttons[1].click()
+        self.assertEqual(self.widget.kernel_eq, self.widget.kernels[1][1])
+        self.widget.kernel_box.buttons[2].click()
+        self.assertEqual(self.widget.kernel_eq, self.widget.kernels[2][1])
+        self.widget.kernel_box.buttons[3].click()
+        self.assertEqual(self.widget.kernel_eq, self.widget.kernels[3][1])
+
+    # Check if right spinner box for selected kernel are visible after widget start
+    @unittest.skip("Not corrected widget on load")
+    def test_kernel_display_run(self):
+        for button_pos, value in ((0, [False, False, False]),
+                                  (1, [True, True, True]),
+                                  (2, [True, False, False]),
+                                  (3, [True, True, False])):
+            if self.widget.kernel_box.buttons[button_pos].isChecked():
+                self.assertEqual([not self.widget._kernel_params[i].box.isHidden() for i in range(0, 3)], value)
+                break
+
+    # Check if right spinner box for selected kernel are visible after we select kernel
+    def test_kernel_display(self):
+        for button_pos, value in ((0, [False, False, False]),
+                                  (1, [True, True, True]),
+                                  (2, [True, False, False]),
+                                  (3, [True, True, False])):
+            self.widget.kernel_box.buttons[button_pos].click()
+            self.widget.kernel_box.buttons[button_pos].isChecked()
+            self.assertEqual([not self.widget._kernel_params[i].box.isHidden() for i in range(0, 3)], value)
+
+    # Check if both spinner box is visible after starting widget
+    def test_optimization_box_visible(self):
+        self.assertEqual(self.widget._optimization_params[0].box.isHidden(), False)
```

Comment:
Starting identifiers with an underscore implies that the interface is protected/hidden and should not be used from outside the class' methods.